tools/nvqpp/nvq++.in

```diff
@@ -39,10 +39,28 @@ function add_pass_to_pipeline {
 }
 
 function list_targets {
-	ls -I *.cpp -1 ${install_dir}/targets/ | sed -e 's/\.config$//'
+	ls -I *.cpp -1 ${install_dir}/targets/ | grep ".config$" | sed -e 's/\.config$//'
```

Comment:
Should we be using `basename` here?

Reply:
Without the additional `grep`, non-targets such as `mapping` are listed. `basename` invocation will require a loop.

Reply:
```bash
find ${install_dir}/targets -name '*.config' -print0 | xargs -0 -L -exec basename {} .config \;
```
Another way ... should work with filename with spaces?

Reply:
This throws an error with 'xargs'.